### PR TITLE
Fixing up CHANGELOG

### PR DESCRIPTION
**Pull request by richardjgowers**
CHANGELOG took a battering, this fixes it

Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: package/CHANGELOG [hunk 1]
```diff
@@ -13,9 +13,9 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-mm/dd/17 richardjgowers, rathann, jbarnoud, orbeckst
+mm/dd/17 richardjgowers, rathann, orbeckst
```
Comment (kain88-de) [inline package/CHANGELOG]: I thought me and @utkbansal are the only two people with commits in the release right now

Comment (richardjgowers) [inline package/CHANGELOG]: Oliver did some work on the TRZ fix, I've done some work on merging etc?

**Comment by richardjgowers** [comment]
I think the only really important change is to put @utkbansal into the 16.2 notes, everything else will get fixed later

File: package/CHANGELOG (hunk 1 above)
Comment (kain88-de) [inline package/CHANGELOG]: Well it's not like we are not adding any more commits to develop anyway.

**Pull request merged**